### ISSUE TITLE
Push a new version

### ISSUE THREAD

**Issue by julien-rodrigues** [brentvatne/react-native-overlay#56]
Hi guys,

I saw you pushed a fix for support of the new versions of React Native but... on master.
And not on npm.

Is there a particular reason for that?

**Comment by brunolemos** [comment]
@brentvatne

**Comment by vinch** [comment]
+1

**Comment by brentvatne** [comment]
I don't use this library anymore :) If anyone wants to take it over, let me know

**Comment by brentvatne** [comment]
OK I pushed 0.5.0 from latest master, and tweeted this: https://twitter.com/notbrent/status/779001989862608897

**Comment by vinch** [comment]
@brentvatne Fair enough. Do you know any alternatives? Thanks for the great work anyway, I think I'm using at least 5 of your libraries.
